tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -124,55 +123,233 @@ def get_virtctl():
+
+        for cluster in get_non_acm_cluster_config():
+            config.switch_ctx(cluster.MULTICLUSTER["multicluster_index"])
+            namespace = config.ENV_DATA["cluster_namespace"]
+            cluster_name = config.ENV_DATA.get("cluster_name")
+
+            try:
+                sc_ocp = ocp.OCP(kind=constants.STORAGECLASS)
+                if sc_ocp.is_exist(resource_name=sc_name):
+                    log.info("Deleting SC %s on %s", sc_name, cluster_name)
+                    delete_storageclass_and_deregister(sc_name=sc_name, sc_ocp=sc_ocp)
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)